packages/meditrak-server/src/routes/authenticate.js

```diff
@@ -130,16 +159,26 @@ const checkAuthentication = async req => {
 export async function authenticate(req, res) {
   await req.assertPermissions(allowNoPermissions);
 
-  const { refreshToken, user, accessPolicy } = await checkAuthentication(req);
+  const { refreshToken, user, accessPolicy } = await checkUserAuthentication(req);
+  const {
+    user: apiClientUser,
+    accessPolicy: apiClientAccessPolicy,
+  } = await checkApiClientAuthentication(req);
+
+  const mergedAccessPolicy = apiClientUser
```

Comment:
The idea of merging access policies is a bit odd, so I think it's worth comments in a few places like this
```suggestion
  // Response access policy contains all permissions for the base API client, plus the logged in user
  const mergedAccessPolicy = apiClientUser
```